Fixes multiline markdown link text (#16317)
mdn/content: files/en-us/web/api/baseaudiocontext/decodeaudiodata/index.md

playing, and stop it playing, respectively. When the `stop()` method is
called on the source, the source is cleared out.

> > **Note:** You can [run the example live](https://mdn.github.io/webaudio-examples/decode-audio-data/) and acces the [source code](https://github.com/mdn/webaudio-examples/tree/master/decode-audio-data).

Reviewer: ```suggestion
> **Note:** You can [run the example live](https://mdn.github.io/webaudio-examples/decode-audio-data/) and access the [source code](https://github.com/mdn/webaudio-examples/tree/master/decode-audio-data).
```

playing, and stop it playing, respectively. When the `stop()` method is
called on the source, the source is cleared out.

> > **Note:** You can [run the example live](https://mdn.github.io/webaudio-examples/decode-audio-data/) (or [view the source](https://github.com/mdn/webaudio-examples/tree/master/decode-audio-data).)

Reviewer: ```suggestion
> **Note:** You can [run the example live](https://mdn.github.io/webaudio-examples/decode-audio-data/) and acces the [source code](https://github.com/mdn/webaudio-examples/tree/master/decode-audio-data).
```
